docs/topics/settings.rst

```diff
@@ -275,6 +275,27 @@ project name). This name will be used for the logging too.
 It's automatically populated with your project name when you create your
 project with the :command:`startproject` command.
 
+.. setting:: CHECK_SETTINGS_ENABLED
+
+CHECK_SETTINGS_ENABLED
+----------------------
+
+Default: ``True``
+
+Whether to enable Check Settings.
+
+.. setting:: CHECK_SETTINGS_MIN
+
+CHECK_SETTINGS_MIN
+------------------
+
+Default: ``70``
```

Comment:
I wonder if this is worth parameterizing. No strong opinion either way, though.

Reply:
I wasnt sure about this, will delete it.